django_better_admin_arrayfield/forms/fields.py

```diff
@@ -2,12 +2,15 @@
 
 from django import forms
 from django.contrib.postgres.utils import prefix_validation_error
+from django.utils.translation import gettext_lazy as _
 from django_better_admin_arrayfield.forms.widgets import DynamicArrayWidget
 
 
 class DynamicArrayField(forms.Field):
 
-    default_error_messages = {"item_invalid": "Item %(nth)s in the array did not validate: "}
+    default_error_messages = {
+        "item_invalid":  _("Item %(nth)s in the array did not validate: "),
```

Comment:
wrongly formatted

Reply:
After reformat with black shows me that this line is properly reformatted. What version of  `black` you are using?